src/_kw

```diff
@@ -1,7 +1,8 @@
 #compdef kw
 
 # Main dispatcher
-_kw() {
+_kw()
+{
```

Comment:
I add a comment about this in the previous commit; this commit should not exist. Go back to the old commits, and make a good change there.
Take a look at the command: `git rebase -i`.